src/main/webapp/WEB-INF/appengine-web.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Nit: If sticking with 4 spacing, then format this file to conform as well.

Reply:
Sure!